chrome/content/zotero/integration/quickFormat.js

```diff
@@ -1650,6 +1701,19 @@ var Zotero_QuickFormat = new function () {
 		return { lastBubble: lastBubble, startOfTheLine: startOfTheLine };
 	}
 	
+	// Make window draggable when mouse is over the dialog's wrapper
+	function _onDialogMouseOver(e) {
+		dialog.style["-moz-window-dragging"] = e.target == dialog ? "drag" : "no-drag";
+	}
```

Comment:
Why do this instead of
```css
.entry {
    -moz-window-dragging: drag;
}

.deck {
    -moz-window-dragging: no-drag;
}
```
And then avoid dragtracker?

Reply:
Oh yeah, no reason to not do that. Thank you. I was trying a bunch of different things trying to figure out why the drag wouldn't start closer to the edges of the window - that's where this came from